src/pc/PlatformDeviceControl.c

```diff
@@ -564,6 +580,46 @@ int usbPlatformConnect(const char *devPathRead, const char *devPathWrite, void *
 #endif  /*USE_USB_VSC*/
 }
 
+int tcpipPlatformConnect(const char *devPathRead, const char *devPathWrite, void **fd)
+{
+#if defined(USE_TCP_IP)
+    sockfd = socket( AF_INET, SOCK_STREAM, 0 );
+    if( sockfd < 0 )
+    {
+        perror("socket");
+        return -1;
+    }
+
+    struct sockaddr_in serv_addr;
+    memset( &serv_addr, 0, sizeof( serv_addr ) );
+
+    int len = strlen( devPathWrite );
+    char devPathWriteBuff[len];
+    strcpy( devPathWriteBuff, devPathWrite );
+
+    char* serv_ip = strtok( devPathWriteBuff, ":" );
+    char* serv_port = strtok( NULL, ":" );
```

Comment:
If a library is found for networking operations, this could be performed by it as well.

Reply:
not sure if I understand this. can help elaborate more?

Reply:
URL (URI) parsing is common operation regarding networking and can usually be found in these kind of libraries as well.
If that's the case, then these library functions can be used instead. 
Random example:
```
url u("http://stackoverflow.com/questions/2616011/parse-a.py?url=1");
cout << u.protocol() << '\t' << u.host() << ...
```